packages/e2e-tests/config/setup-debug-artifacts.js

```diff
@@ -1,54 +0,0 @@
-/**
```

Comment:
Why was it removed?

Reply:
haha, this makes it harder to know the exact reasons of a failing test locally, I'll restore it.